Tests.Integration.Presentation.Web/ItSystem/ItSystemInterfacesTest.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Core.DomainModel;
+using Core.DomainModel.ItSystem;
+using Core.DomainModel.Organization;
+using Presentation.Web.Models;
+using Tests.Integration.Presentation.Web.Tools;
+using Xunit;
+
+namespace Tests.Integration.Presentation.Web.ItSystem
+{
+    public class ItSystemInterfacesTest : WithAutoFixture
+    {
+        [Theory]
+        [InlineData(OrganizationRole.GlobalAdmin)]
+        public async Task Global_Administrator_Can_Get_All_Interfaces(OrganizationRole role)
+        {
+            const string interFacePrefixName = "GlobalAdminInterfaceTest";
```

Comment:
Det er ikke unikt for din test case.. det er konstant. Benyt en unik bestanddel f.eks. A<Guid>().ToString("N")